robot: 
  #arm_name: arm
  arm_name: arm  #moveit2_tutorials demo.launch.py
  ee_link_name: panda_link8
  robot_desc: move_group robot_description
  planning_scene: planning_scene
  planning_frame: world
  num_cart_pts: 20
topic:
  sub:
    cmd_pose:
      name: arm/command/cmd_pose
      queue: 1
    cmd_tool_orientation:
      name: tool/command/cmd_orientation
      queue: 1
    cmd_delta_pose: 
      name: arm/command/cmd_delta_pose
      queue: 1  
  pub:
    display_trajectory:
      name: arm/display_path
      queue: 1
    current_pose:
      name: arm/state/current_pose
      queue: 1
srv: 
  disable_collision: 
    name: /move_group/disable_collision
  add_collision: 
    name: /move_group/add_collision
  start_position_ctl: 
    name: start_position_ctl
  start_joint_traj_ctl: 
    name: start_joint_traj_ctl
  start_joint_group_pos_ctl: 
    name: start_joint_group_pos_ctl
  start_joint_group_vel_ctl: 
    name: start_joint_group_vel_ctl
  change_robot_state: 
    name: change_state